pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -258,19 +257,19 @@ func (m *ManagerImpl) Start(activePods ActivePodsFunc, sourcesReady config.Sourc
 	}
 	if selinux.SELinuxEnabled() {
 		if err := selinux.SetFileLabel(m.socketdir, config.KubeletPluginsDirSELinuxLabel); err != nil {
-			klog.Warningf("Unprivileged containerized plugins might not work. Could not set selinux context on %s: %v", m.socketdir, err)
+			klog.InfoS("Unprivileged containerized plugins might not work. Could not set selinux context on %s: %v", "path", m.socketdir, "err", err)
 		}
 	}
 
 	// Removes all stale sockets in m.socketdir. Device plugins can monitor
 	// this and use it as a signal to re-register with the new Kubelet.
 	if err := m.removeContents(m.socketdir); err != nil {
-		klog.Errorf("Fail to clean up stale contents under %s: %v", m.socketdir, err)
+		klog.ErrorS(err, "Fail to clean up stale content under socket dir", "path", m.socketdir)
 	}
 
 	s, err := net.Listen("unix", socketPath)
 	if err != nil {
-		klog.Errorf(errListenSocket+" %v", err)
+		klog.ErrorS(err, errListenSocket)
```

Comment:
```suggestion
		klog.ErrorS(err, "Failed to listen to socket while starting device plugin registry")
```
lets in-line the error message as done above for `errListAndWatch` ?
cc: @serathius